Lux/src/lux_web.cpp

```diff
@@ -416,10 +442,40 @@ void set_slider_value( std::string name, float value ) {
         any_function& fn = global_context->s->functions[ name ];
         if( std::holds_alternative< any_fn< float > >( fn ) ) {
             global_context->s->get_fn_ptr< float, slider_float >( name )->value = value;
+            
+            global_context->s->ui.displayed = false;
+            
+            if( global_context->s->ui.running && (
+                name.find("start") != std::string::npos || 
```

Comment:
We should not be looking for scene specific names in backend. Instead, feed audio into functor and use scene file to specify which sliders change.